include/OpenSpaceToolkit/Astrodynamics/Flight/System/Dynamics.hpp

```diff
@@ -61,21 +61,35 @@ class Dynamics
 
     virtual void print(std::ostream& anOutputStream, bool displayDecorator = true) const = 0;
 
-    /// @brief              Obtain dynamical equations function wrapper (pure virtual)
+    /// @brief              Get dynamical equations function wrapper (pure virtual)
     ///
     /// @return             std::function<void(const std::vector<double>&, std::vector<double>&, const double)>
 
+    static DynamicalEquationWrapper getDynamicalEquations(
+        const Instant& anInstant, const Array<Shared<Dynamics>>& aDynamicsArray
+    );
+
+    /// @brief              Update the state derivative (pure virtual)
+    ///
+    /// @param              [in] x A state vector
+    /// @param              [in] dxdt A state derivative vector
+    /// @param              [in] anInstant An instant
+
     virtual void update(const StateVector& x, StateVector& dxdt, const Instant& anInstant) = 0;
```

Comment:
Thoughts on naming here? Fine by me but just want to be sure it conveys the right concept

Reply:
Made a poll